src/dmd/mtype.h

```diff
@@ -41,6 +41,11 @@ typedef struct TYPE type;
 
 void semanticTypeInfo(Scope *sc, Type *t);
 
+#if IN_LLVM
```

Comment:
Why this? Does GDC need this, @ibuclaw ?

Reply:
Hum I'll remove those commits from the `stable` PR. I just noticed it was exposed but not in the header, yes LDC use it. I'm surprised it wasn't submitted to upstream, aren't you guys trying to reduce your diff anymore ?

Reply:
I am under the impression that they gave up on that, there are enough people working on ldc that the maintenance burden is low vs gdc's one man show where it is my interest to do as little work as possible.

Reply:
You don't need to have this in an #ifdef, if it's extern C++, then it should be in the headers anyway. And no, I doubt gdc runs typeSemantic anywhere. In the few types that we do build internally, the semantic is completed inline.

Reply:
>  I am under the impression that they gave up on that, there are enough people working on ldc that the maintenance burden is low vs gdc's one man show where it is my interest to do as little work as possible.

https://github.com/ldc-developers/ldc/graphs/contributors?from=2019-01-01&to=2020-04-03&type=c